Makefile

```diff
@@ -10,13 +10,15 @@ test: dev check
 
 check:
 	isort -rc -c fonduer/
```

Comment:
Even if we don't enforce flake8, add style checks for the tests.

Reply:
👍